src/pages/VideoDetail.jsx

```diff
@@ -1,16 +1,21 @@
-import { Link, useParams } from "react-router-dom";
-import { useVideoData } from "../custom-hooks";
+import { SideNav } from "../components";
 import { useEffect, useState } from "react";
+import { useVideoData } from "../custom-hooks";
+import { Link, useParams } from "react-router-dom";
 import { useGenericData, useAuth } from "../contexts";
-import { SideNav } from "../components";
-import { addToWatchLater } from "../helper-functions";
+
+import {
+  addToWatchLater,
+  addToLikes,
+  removeFromLikes,
+} from "../helper-functions";
 
 function VideoDetail() {
   const { user } = useAuth();
   const { vid } = useParams();
   const videoData = useVideoData();
   const [videoObj, setObj] = useState(null);
-  const { boolFunc } = useGenericData();
+  const { state, boolFunc } = useGenericData();
```

Comment:
you could have given a better name instead of `boolFunc`, basically a function name should describe what the function will do